### ISSUE TITLE
[web] Uncaught ReferenceError: __DEV__ is not defined

### ISSUE THREAD

**Issue by ethanneff** [th3rdwave/react-native-safe-area-context#253]
```
Uncaught ReferenceError: __DEV__ is not defined
    at Module../node_modules/react-native-safe-area-context/lib/module/SafeAreaContext.js 
```

![Screen Shot 2022-03-20 at 7 54 00 PM](https://user-images.githubusercontent.com/2933593/159199406-1b0f852b-9299-4f29-ac7a-e7c9bc4adf1c.png)

**Comment by ethanneff** [comment]
Downgrading to 3.4.1 solves the issue

**Comment by janicduplessis** [comment]
Hey thanks for reporting, should be fixed in 4.2.2

**Comment by ethanneff** [comment]
Thank you for the quick fix 💯 !
